indico/MaKaC/webinterface/tpls/ConfDetailsFull.tpl

```diff
@@ -63,6 +63,8 @@
   </div>
   % endif
 
+  ${ template_hook('survey-announcement', event=conf) }
```

Comment:
nope, you can use it multiple times and even specify a priority in the `@template_hook()` call if you care about the order relative to other functions using that hook